scripts/generate-repos-info.js

```diff
@@ -0,0 +1,71 @@
+const fs = require("fs");
+const axios = require("axios");
+const listr = require("listr")
+
+axios.defaults.timeout = 5000;
+
+const axiosConfig = {
+  headers: {
+    "content-type": "application/json",
+    Accept: "application/vnd.github.v3+json"
+  },
+}
+
+const repoList = require("../website/config/docs").map(v => v.githubRepo)
+const res = {}
+
+const tasks = new listr([
+  {
+    title: "Fetch repos' info and good first issues",
+    task: ()=> new listr(repoList.map((repo)=>{
+      return {
+        title: `Fetch ${repo}`,
+        task: ()=>new listr([
+          {
+            title: `${repo}'s info`,
+            task: () => axios.get(`https://api.github.com/repos/${repo}`, axiosConfig)
```

Comment:
We should consider the case of a failed data pull, at which point we might be able to interrupt the CI execution.

Reply:
I use this part.
```js
tasks.run()
  .then(()=> {
    console.log("[Finish] Generate repos' info and good first issues finished")
  })
  .catch((err)=>{
    console.error(err)
    process.exit(1)
  })
```
Is it not enough? I'm not familiar with this part. I'll check other files.

Reply:
Yes, I think so. When exiting with a non-zero status code should cause the CI to report an error and interrupt.